src/System.Net.Sockets/tests/FunctionalTests/ReceiveMessageFrom.cs

```diff
@@ -41,5 +41,111 @@ public void Success()
                 }
             }
         }
+
+        [OuterLoop] // TODO: Issue #11345
+        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsSubsystemForLinux))] // https://github.com/Microsoft/BashOnWindows/issues/987
+        public void Success_IPv6()
+        {
+            if (Socket.OSSupportsIPv6)
```

Comment:
Maybe OSSupportsIPv6 should factor in IsNotWindowsSubsystemForLinux, and then the ConditionalFact could be based on OSSupportsIPv6 instead, so that we don't have this if block and so that the test will show up as skipped if IPv6 isn't supported?